tests/files/tests.py

```diff
@@ -169,6 +169,25 @@ def test_io_wrapper(self):
             test_file.seek(0)
             self.assertEqual(test_file.read(), (content * 2).encode())
 
+    @unittest.skipIf(os.name == 'nt', "Posix specific tests.")
```

Comment:
Do we need to skip on other OS?

Reply:
Don't know exactly, I skipped them because we just changed `lock/unlock` for posix and I tested them on `Linux`.
Let's remove the `skipIf` and run them on all OS to check what will happen.